### PR TITLE
Fix Codacy badges

### PR DESCRIPTION
**Pull request by vladimyr**
Refactor & support new default git branch naming convention.

### CODE REVIEW

**Comment by amio** [comment]
Seems two examples are broken https://badgennet-git-codacy.badgen.vercel.app/codacy
image

While I forgot which project this id (`f0875490cea1497a9eca9c25f3f7774e `) refers to >.<

**Comment by vladimyr** [comment]
Yes they are 🙁 I would fix them but couldn't figure out how to get repo URL from Codacy id 🤷

File: api/codacy.ts [hunk 1]
```diff
@@ -34,40 +33,39 @@ const SUBJECT_BY_TYPE = {
   grade: 'code quality'
 }
 
-async function handler ({ type, projectId, branch = 'master' }: PathArgs) {
-  if (projectId) {
-    const svg = await got(`${uriBase}/${type}/${projectId}`,{
-      searchParams: { branch: branch && branch.replace('-', '--') }
```
Comment (vladimyr) [inline api/codacy.ts:40]: @amio I have no idea where did this come from? :confused: Reading through [docs](https://docs.codacy.com/repositories/badges/) I haven't seen mention of branch name _escaping_ :confused:

Comment (amio) [reply api/codacy.ts:40]: I don't know either = =

Guess we need to find another repo to demonstrate this feature
Found this one https://github.com/codacy/codacy-coverage-reporter (1c524e61cd8640e79b80d406eda8754b)

Comment (vladimyr) [reply api/codacy.ts:40]: @amio We won't get far with that one too :tired_face: 

- default branch coverage: 
- default branch grade: 
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch coverage: 
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch grade: 

<details>
<summary>View source</summary>

```md
- default branch coverage: ![](https://api.codacy.com/project/badge/coverage/1c524e61cd8640e79b80d406eda8754b)
- default branch grade: ![](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b)
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch coverage: ![](https://api.codacy.com/project/badge/coverage/1c524e61cd8640e79b80d406eda8754b?branch=script-tests)
- [`script-tests`](https://github.com/codacy/codacy-coverage-reporter/tree/script-tests) branch grade: ![](https://api.codacy.com/project/badge/grade/1c524e61cd8640e79b80d406eda8754b?branch=script-tests)
```
</details>

Comment (vladimyr) [reply api/codacy.ts:40]: @amio I figured out which project was originally used: https://github.com/xobotyi/react-scrollbars-custom but branch badges still don't work :shrug: 

- default branch coverage: 
- default branch grade: 
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch coverage: 
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch grade: 

<details>
<summary>View source</summary>

```md
- default branch coverage: ![](https://api.codacy.com/project/badge/coverage/f0875490cea1497a9eca9c25f3f7774e)
- default branch grade: ![](https://api.codacy.com/project/badge/grade/f0875490cea1497a9eca9c25f3f7774e)
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch coverage: ![](https://api.codacy.com/project/badge/coverage/f0875490cea1497a9eca9c25f3f7774e?branch=dev-master)
- [`dev-master`](https://github.com/xobotyi/react-scrollbars-custom/tree/dev-master) branch grade: ![](https://api.codacy.com/project/badge/grade/f0875490cea1497a9eca9c25f3f7774e?branch=dev-master)
```
</details>

Comment (amio) [reply api/codacy.ts:40]: I guess we have to get rid of the branch example for now  =.=

Comment (vladimyr) [reply api/codacy.ts:40]: Or we could kinda cheat by using `?branch=master` as shields examples do 🤷

Comment (amio) [reply api/codacy.ts:40]: That's not bad 😂

Comment (vladimyr) [reply api/codacy.ts:40]: Done :heavy_check_mark:

**Comment by amio** [comment]
Thanks! @vladimyr

**Pull request merged**